### ISSUE TITLE
Emit per stream metrics - bit rate, resolution

### ISSUE THREAD

**Issue by Strykar** [livepeer/go-livepeer#2200]
**Is your feature request related to a problem? Please describe.**
O's wish to measure concurrent stream count + what resolution and bit rate each was transcoded at.
This is critical to plan infrastructure growth.
We charge per pixel, we should be able to measure how many total pixels (according to LPMS / Mist) we have transcoded.

**Describe the solution you'd like**
Emit Prometheus metrics that let us track concurrent streams by resolution and bit rate.

**Comment by papabear99** [comment]
Can Broadcaster IP addresses be added to the stream metrics so Orchestrators can see where their work is originating?  

Having this information available in Prometheus/Grafana would help Orchestrators plan their infrastructure expansion.

**Comment by hthillman** [comment]
does this address the concerns here? https://github.com/livepeer/go-livepeer/pull/2238

**Comment by papabear99** [comment]
Looks like it takes care of the pixel count and broadcast IP requests.  

Doesn't look like bit rate is addressed. 

@Strykar are you looking for full resolution i.e. 720x480p per stream vs. the current horizontal only resolution i.e. 480p displayed in the session profile?

**Comment by darkdarkdragon** [comment]
@papabear99 @Strykar 
Pixels/second is the only metrics that shows real load.

For resolution - one stream could have 2 target renditions, other stream 4. How would you group this in metrics?

**Comment by Strykar** [comment]
@darkdarkdragon I've asked @thulinma to chime in here

**Comment by hthillman** [comment]
I'm inclined to agree with @darkdarkdragon's assessment here

**Comment by Thulinma** [comment]
Yeah it looks like the `mil_pixels_processed` stat is the best one to track actual load on the system over time.

I can see use for having buckets of input/output resolutions handled (per video track in/out), too - for tracking the kind of capabilities hardware needs to have to handle average requests. Unfortunately I'm not deep enough into go-livepeer's code yet to know exactly how the prometheus metrics work; how do those eventually get created / in what format? Or is that always done by some stand-alone tool next to go-livepeer, perhaps? It would also make sense to not log every client IP there, but instead group by geographical location through for example a 3-character geohash.

**Comment by papabear99** [comment]
I'd like to see the `mil_pixels_processed` metric added to the benchmark and then have the option to set -maxPixels instead of `-maxSessions` since it's a better measure of load than sessions.

IMO now that we the metric it would be nice to give it more value and have a real use case for it besides just being able to view the metric.
![PixelCounters](https://user-images.githubusercontent.com/89408276/156269232-e64ccab7-eb82-4d57-9cee-ac0eaf90e1b9.jpg)
 
@[stronk-dev](https://github.com/stronk-dev)
Used the `-metricsClientIP` metric to make a Grafana panel that shows pixels processed by region but I think pixels processed better used as a throughput measure and seems a bit odd to have tied to the `mil_pixels_processed` metric.
![PixelPerRegion](https://user-images.githubusercontent.com/89408276/156269788-103453ec-12d2-4f11-aa85-684b2fb0781f.png)
.

**Comment by hthillman** [comment]
I wonder if ` buckets of input/output resolutions` is an area where a community member might be able to help?

could probably use Ivan's PR as a reference
https://github.com/livepeer/go-livepeer/pull/2238/files
